encoding/ccf/consts.go

```diff
@@ -0,0 +1,159 @@
+/*
+ * Cadence - The resource-oriented smart contract programming language
+ *
+ * Copyright Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ccf
+
+// CCF uses CBOR tag numbers 128-255, which are unassigned by [IANA]
+// (https://www.iana.org/assignments/cbor-tags/cbor-tags.xhtml).
+//
+// !!! *WARNING* !!!
+//
+// CCF Codec *MUST* comply with CCF Specifications.  Relevant changes
+// must be in sync between codec and specifications.
+//
+// Only add new tag number by:
+// - replacing existing placeholders (`_`) with new tag number
```

Comment:
By not appending to the end, we can have larger block of unused tag numbers if we ever need it.

Tag numbers in 128-255 are unassigned in CBOR, and we currently use 128-231.  Since each group of tags in this range have reserved space available, there is no need to append new tag numbers in 232-255.

Reply:
I see, that makes sense. Could you please add that explanation as a comment to the code?

Reply:
Done in commit 4a6aa577f9b69074f1dfc0e20743031e73e1486a.